src/components/Card/Card.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import S from "./Card.module.css";
+
+import { sampleFolderData } from "@src/api/api";
+import { SampleFolder } from "@src/type/type";
+import displayCreateAt from "./components/displayCreateAt";
+
+const Card = () => {
+  const [card, setCard] = useState<SampleFolder>();
+
+  useEffect(() => {
+    async function fetchData() {
+      const data = await sampleFolderData();
+      setCard(data);
+    }
+    fetchData();
+  }, []);
+
+  // links.createdAt yyyy.mm.dd 형태로 변환
+  const createDate = (date: string): string => {
```

Comment:
P2:
어떤 일을 하는 함수인지 알 수 있게 네이밍해주시면 더 좋겠습니다.

```suggestion
  const formatCreateAt = (date: string): string => {
```

Reply:
만약 이런식으로 yyyy.mm.dd 형식으로 변환하는 일이 반복된다면 해당 함수를 반복사용할 수 있게 분리해도 좋을 것 같습니다.